python/ccxt/phemex.py

```diff
@@ -128,6 +128,7 @@ def describe(self):
                         'exchange/wallets/confirm/withdraw',  # ?code=<withdrawConfirmCode>
                         'exchange/wallets/withdrawList',  # ?currency=<currency>&limit=<limit>&offset=<offset>&withCount=<withCount>
                         'exchange/wallets/depositList',  # ?currency=<currency>&offset=<offset>&limit=<limit>
+                        'exchange/wallets/v2/depositAddress',  # ?currency=<currency>
```

Comment:
Can you plz move this edit to the JS file?

Reply:
Sorry, my mistake. I have forgot. Sure, I will do it.

Reply:
@docek thx so much!